src/test/resources/com/puppycrawl/tools/checkstyle/checks/descendanttoken/InputDescendantTokenIllegalTokens4.java

```diff
@@ -20,7 +20,9 @@
     public void methodWithPreviouslyIllegalTokens()
     {
         int i = 0;
-        switch (i) // violation
+        // violation below 'Count of 1 for 'LITERAL_SWITCH' descendant
+        // 'LITERAL_DEFAULT' is less than minimum count 2.'
```

Comment:
Why this line is required?
Put message to single line, compact it with regexp if required

Reply:
On single line it doesn't fit with 100 chars limit, even with regexp, 

```
        switch (i) // violation below 'C.*t of 1 for '.*_SWITCH' des.*t '.*_DEFAULT' is less than min .* 2.'
```

so I am thinking about take that line above

Reply:
@romani,

```
 32:12: Count of 1 for 'LITERAL_NATIVE' descendant 'LITERAL_NATIVE' exceeds maximum count 0.
```

In above statement 32 is a line number but what is 12 ?

Reply:
Try it like:
`'Count of 1 .* LITERAL_DEFAULT'' is less than minimum count 2'`